analyses/molecular-subtyping-embryonal/02-generate-subset-files.R

```diff
@@ -0,0 +1,101 @@
+# Stephanie J. Spielman and Jaclyn Taroni for ALSF CCDL 2020
+#
+# This script subsets the files required for subtyping non-MB and non-ATRT
+# embryonal tumors. A sample will be included if either of the following
+# conditions are met: 1) the sample is labeled as an embryonal tumor
+# (broad_histology) but NOT an MB or ATRT tumor (disease_type_old) OR 2)
+# the sample contains a TTYH1 fusion (5' partner)
+
+library(tidyverse)
+
+#### Directories ---------------------------------------------------------------
+
```

Comment:
Maybe add a comment here about what the `rprojroot::find_root` function is doing. This may not be so intuitive for someone who has not been following the project.